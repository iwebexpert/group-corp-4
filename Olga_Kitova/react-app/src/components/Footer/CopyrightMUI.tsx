import React from 'react'
import Typography from '@mui/material/Typography'
import Link from '@mui/material/Link'
import { TypographyProps } from '@mui/system'

export default function CopyrightMUI(props: TypographyProps) {
  return (
    <Typography variant="body2" color="text.secondary" align="center" {...props}>
      {'Copyright Â© '}
      <Link color="primary" href="#">
        CMS
      </Link>{' '}
      {new Date().getFullYear()}
      {'.'}
    </Typography>
  )
}
