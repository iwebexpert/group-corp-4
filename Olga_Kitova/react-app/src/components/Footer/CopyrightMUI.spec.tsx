import React from 'react'
import * as renderer from 'react-test-renderer'
import { test, expect } from '@jest/globals'
import CopyrightMUI from './CopyrightMUI'

test('Testing with snapshots (CopyrightMUI)', () => {
  const testRenderer = renderer.create(<CopyrightMUI />)
  const treeJson = testRenderer.toJSON()
  expect(treeJson).toMatchSnapshot()
})
