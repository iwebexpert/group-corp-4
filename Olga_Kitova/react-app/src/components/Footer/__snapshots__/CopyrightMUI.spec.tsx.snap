// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing with snapshots (CopyrightMUI) 1`] = `
<p
  className="MuiTypography-root MuiTypography-body2 MuiTypography-alignCenter css-1vl95ex-MuiTypography-root"
>
  Copyright Â© 
  <a
    className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1iwtne7-MuiTypography-root-MuiLink-root"
    href="#"
    onBlur={[Function]}
    onFocus={[Function]}
  >
    CMS
  </a>
   
  2022
  .
</p>
`;
